app/src/main/java/com/github/fribourgsdp/radio/Playlist.kt

```diff
@@ -78,4 +78,12 @@ data class Playlist (var name: String, var genre: Genre) {
     fun getSongs(): Set<Song> {
         return songs.toSet()
     }
+
+    override fun hashCode(): Int {
+        return name.hashCode()
+    }
+
+    override fun equals(other: Any?): Boolean {
+        return name == (other as Playlist).name
+    }
```

Comment:
Useless because Playlist is a data class

Reply:
Not so useless because we want to be able to exclude playlist from sets (HashSet/TreeSet) based only on their name, so that no two playlists with the same name are in that set.

Reply:
Yes indeed you are absolutely right, my bad